src/RFC-0205_LedgerTransactions.md

```diff
@@ -0,0 +1,148 @@
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+## Language
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT
+RECOMMENDED", "MAY" and "OPTIONAL" in this document are to be interpreted as described in
+[BCP 14](https://tools.ietf.org/html/bcp14) (covering RFC2119 and RFC8174) when, and only when, they appear in all
+capitals, as shown here.
+
+## Disclaimer
+
+This document and its content are intended for information purposes only and may be subject to change or update without
+notice.
+
+This document may include preliminary concepts that may or may not be in the process of being developed by the Tari
+community. The release of this document is intended solely for review and discussion by the community of the
+technological merits of the potential system outlined herein.
+
+## Goals
+
+This Request for Comment (RFC) presents a design for a hardware wallet with MimbleWimble
+
+## Related Requests for Comment
+
+- [RFC-0201: TariScript](RFC-0201_TariScript.md)
+- [RFC-0203: Stealth Address](RFC-0203_StealthAddress.md)
+
+## Introduction
+
+Hardware wallets are secure physical devices used to protect crypto assets and funds by requiring user interaction from the device.
+These air-gapped devices are used to sign for transactions by keeping all the secrets from the transactions on the hardware device. If
+the host machines that run the wallets are compromised by malware the secrets from the transactions are still safe as they are not
+kept on the machine as with regular wallets. The devices are very low power and only feature a very limited processing power. MimbleWimble 
+has intensive crypto operations that are in some instances very slow or not able to run on the hardware wallet at all. This RFC describes a way to 
+get around these limitations to have fully functional secure hardware wallet integration. 
+
+## Background
+
+Vanilla [Mimblewimble] only has a single secret per [UTXO], the blinding factor (\\( k_i \\) ), the Tari protocol extends the [Mimblewimble] protocol to include scripting in the form of [TariScript]. This adds a second secret per [UTXO] called the script_key (\\( k_s \\) ). In practical terms this means that while vanilla [Mimblewimble] only requires that a wallet wishing to spend an [UTXO], prove knowledge of (\\( k_i \\) ) by singing the kernel signature, this is not sufficient for Tari. A Tari wallet must also prove knowledge of the script key (\\( k_s \\) ), by signing the script signature. 
+
+## Requirements
+
+To properly implement hardware wallets we need the following requirements to be met:
+* No UTXO can be spent without a user physically approving the transaction on the hardware wallet.
+* Users need to verify transaction properties when signing for transactions.
+* The user must be able to receive transactions without having to authorize them on the hardware wallet.
```

Comment:
Umm, with interactive payments, we cannot have a case of a `malicious sender effectively burning funds`. 
The receiver side if wholly constructed by the receiver. 
A malicious receiver is a whole different ball game and I don't think possible to stop

Reply:
Isn't it still possible for a sender to duplicate an existing valid value-and-mask ciphertext with a malformed commitment? Then if the scanner is only defining "receiving" as being able to decrypt the ciphertext, it would be fooled and claim the recipient has funds that in fact cannot be spent.

Reply:
The sender does not construct this, the receiver does. So how can the sender force the receiver to malform its own commitment?

Reply:
Looks like you already made an update later in the RFC to address this. Apologies!